lib/config/presets/internal/monorepo.ts

```diff
@@ -492,6 +492,8 @@ const repoGroups = {
   'trust-dns': 'https://github.com/bluejekyll/trust-dns',
   tsoa: 'https://github.com/lukeautry/tsoa',
   turbo: 'https://github.com/vercel/turbo',
+  'typed-signalr':
```

Comment:
Should it have client in the name?

Reply:
I don't think you need that distinction, the SignalR server is already typed OOB. I have no issue with adding '-client' at the end if you want it to be extra verbose. Let me know what you think.